src/main/java/net/snowflake/client/core/SFTrustManager.java

```diff
@@ -1068,7 +1071,7 @@ private boolean isCached(List<SFPair<Certificate, Certificate>> pairIssuerSubjec
       for (SFPair<Certificate, Certificate> pairIssuerSubject : pairIssuerSubjectList) {
         OCSPReq req = createRequest(pairIssuerSubject);
         CertificateID certificateId = req.getRequestList()[0].getCertID();
-        LOGGER.debug(CertificateIDToString(certificateId));
+        LOGGER.debug(CertificateIDToString(certificateId), true);
```

Comment:
This can be set to `false` as well. In short all Certificate Revocation stuff can be skipped from the secret detector.